components/script/dom/cssstyledeclaration.rs

```diff
@@ -254,27 +254,26 @@ impl CSSStyleDeclaration {
                 }
             };
 
-            // Step 6
+            // Step 5
             let window = self.owner.window();
             let quirks_mode = window.Document().quirks_mode();
-            let result =
-                parse_one_declaration(id, &value, &self.owner.base_url(),
-                                      window.css_error_reporter(),
-                                      PARSING_MODE_DEFAULT,
-                                      quirks_mode);
-
-            // Step 7
-            let parsed = match result {
-                Ok(parsed) => parsed,
+            let mut declarations = SourcePropertyDeclaration::new();
+            let result = parse_one_declaration_into(
+                &mut declarations, id, &value, &self.owner.base_url(),
+                window.css_error_reporter(), PARSING_MODE_DEFAULT, quirks_mode);
+
+            // Step 6
+            match result {
+                Ok(()) => {},
```

Comment:
The spec renumbered (removed an earlier step), and I forgot to renumber steps 8-9 here to 7-8. Done.